src/components/Zoom/Zoom.js

```diff
@@ -110,27 +145,33 @@ function Zoom({
   return (
     // eslint-disable-next-line react/jsx-props-no-spreading
     <div className="rs-zooms-bar" {...other}>
-      <div
-        role="button"
+      <button
+        type="button"
         tabIndex={0}
-        className="rs-zoom-in"
+        className={`rs-zoom-in${
+          disabledControls.zoomIn ? ' rs-zoom-disabled' : ''
```

Comment:
use the disabled attribute, no need of a css class for that.

Reply:
Done